### PR TITLE
added more builds that need to be triggered

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

**Comment by harshad16** [comment]
last week I was thinking of adding this too, however didn't move forward, as once the new release is made for storages, the kebechet instance would try to update the other components and when the update pr is merged in other repo, they would be triggered as well. 
Definitely this is a faster way and might provide reassurance.

**Comment by harshad16** [comment]
@goern I m confused about the `queue` instance though, is calling storages while testing storages a good practice?

**Comment by goern** [comment]
a) why is zuul not gating?!
b) I think this will build all changes between releases

**Comment by goern** [comment]
> @goern I m confused about the `queue` instance though, is calling storages while testing storages a good practice?

chk https://zuul-ci.org/docs/zuul/user/config.html#attr-project.%3Cpipeline%3E.queue

maybe we should put it back in core queue...

File: .zuul.yaml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 - project:
     check:
-      queue: thoth-station/core
+      queue: thoth-station/storages
       jobs:
         - thoth-coala
         - thoth-pytest
     gate:
-      queue: thoth-station/core
+      queue: thoth-station/storages
```
Comment (harshad16) [inline .zuul.yaml]: :)

**Comment by goern** [comment]
regate

**Comment by harshad16** [comment]
> regate
this would, work when we approve it again.

**Comment by harshad16** [comment]
> a) why is zuul not gating?!

I guess, we figured that part out. Now it would work.

> b) I think this will build all changes between releases

Image build trigger for all of those repos for every PR merge of storages doesn't add upto me, as they would only get to know about these changes when there is a release of storages. 

@pacospace  and @fridex  PTAL

**Comment by harshad16** [comment]
@fridex  @pacospace  PTAL :)

**Comment by harshad16** [comment]
recheck

**Pull request merged**